### PR TITLE
correct synfutures v3 tvl data

### PR DESCRIPTION
**Pull request by fan3cn**
bug fix for SynFutures V3 TVL data.

Name (to be shown on DefiLlama):
SynFuturesV3

Twitter Link:
https://twitter.com/SynFuturesDefi

List of audit links if any:
https://www.synfutures.com/Quantstamp-Audit-Report-SynFuturesV3.pdf

Website Link:
https://www.synfutures.com/

Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://github.com/SynFutures/logo/blob/main/logo-svg/SYN-CUBE.svg

Current TVL:
$5.54M

Chain:
Blast

Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
synfutures

Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
1504

Short Description (to be shown on DefiLlama):
SynFutures is a next-generation derivatives exchange focused on creating an open and trustless derivatives market by enabling futures trading on anything, anytime, anywhere. By cultivating a free market and maximizing the variety of tradable assets, SynFutures is lowering the barrier to entry in the derivatives market, creating a more equitable derivatives market.

Token address and ticker if any:
Category (full list at https://defillama.com/categories) *Please choose only one:
Derivatives

Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
PYTH

forkedFrom (Does your project originate from another project):

methodology (what is being counted as tvl, how is tvl being calculated):
The calculation method of TVL is to sum up the dollar value of the balance of erc20 token and native token of all AMM pairs in different networks on a given block.

### CODE REVIEW

**Comment by fan3cn** [comment]
--- arbitrum ---
USDC.e                    0.000644644
WETH                      2.9988314813990004e-8
Total: 0.000644673988314814 

--- blast ---
WETH                      4.14 M
USDB                      1.40 M
Total: 5.54 M 

--- tvl ---
WETH                      4.14 M
USDB                      1.40 M
Total: 5.54 M 

------ TVL ------
arbitrum                  0.000644673988314814
blast                     5.54 M

total                    5.54 M

**Comment by llamatester** [comment]
The adapter at projects/synfutures-v3 exports TVL: 
        
 
 
    arbitrum                  0.0006440300076247399
    blast                     5.56 M
    
    total                    5.56 M

File: projects/synfutures-v3/index.js [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+      const logs = await getLogs({
+        api,
+        target: gate,
+        topics: ['0x2f642d4751a7e76430c4c9b3abe8ab9fd9ed0daa08cae14b5594765049008e02'],
+        fromBlock,
+        eventAbi: 'event NewInstrument(bytes32 index, address instrument, address base, address quote, string symbol, uint total)',
+      })
+      const tokensAndOwners = logs.map(i => ([i.args.quote, i.args.instrument]))
+      tokens.forEach(token => tokensAndOwners.push([token, gate]))
+      return sumTokens2({ api, tokensAndOwners: logs.map(i => ([i.args.quote, i.args.instrument])) })
```
Comment (g1nt0ki) [inline projects/synfutures-v3/index.js:29]: Thanks for the PR, code is mostly good, except for this line, it should be just tokensAndOwners here? Else tokens in gate are skipped?

**Comment by llamatester** [comment]
The adapter at projects/synfutures-v3 exports TVL: 
        
 
 
    arbitrum                  3.73 k
    blast                     8.06 M
    
    total                    8.06 M

File: projects/synfutures-v3/index.js (hunk 1 above)
Comment (fan3cn) [reply projects/synfutures-v3/index.js]: Thanks for the comment. I have corrected the code.

**Pull request merged**